README.md

```diff
@@ -10,8 +10,7 @@
 
 First, install all [build dependencies](docs/dev/dependencies.md).
 
-After cloning this repository, the installer binary will need to be built by running the following:
-
+After cloning this repository to your [GOPATH](https://golang.org/cmd/go/#hdr-GOPATH_environment_variable), and please ensure you have this directory src/github.com/openshift/there to place installer/ in it, the installer binary will need to be built by running the following:
```

Comment:
There's a lot going on in this sentence.  Maybe split clone and build:

```markdown
Clone this repository to `src/openshift/installer` in your [`GOPATH`](https://golang.org/cmd/go/#hdr-GOPATH_environment_variable).
Then build the `openshift-install` binary with:
```

Reply:
> There's a lot going on in this sentence. Maybe split clone and build:
> 
> ```gfm
> Clone this repository to `src/openshift/installer` in your [`GOPATH`](https://golang.org/cmd/go/#hdr-GOPATH_environment_variable).
> Then build the `openshift-install` binary with:
> ```

Much better. Updated it now.